docs/platforms/android/session-replay/index.mdx

```diff
@@ -120,8 +120,12 @@ Errors that happen on the page while a replay is running will be linked to the r
 
 ## FAQ
 
+Q: Why are parts of my replay not masked?
+A: Text fields, input fields, images, video players and webviews are all masked by default. Local assets, such as colors or vector drawables, are not masked due to their low likelihood of containing PII. If you encounter a view/component that should be masked by default, consider opening a [GitHub issue](https://github.com/getsentry/sentry-java/issues).
+
```

Comment:
```suggestion
Q: Why are parts of my replay not masked?
A: Text fields, input fields, images, video players and webviews are all masked by default. Local assets, such as colors or vector drawables, aren't masked because the likelihood of these assets containing PII is low. If you encounter a view/component that should be masked by default, consider opening a [GitHub issue](https://github.com/getsentry/sentry-java/issues).

```